lib/boilerPlate.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const util = require('util');
+
+/**
+ * Error for whatever you want!
+ *
+ * @extends Error
+ * @class Errors.BoilerPlate
+ * */
+class BoilerPlate extends Error{
+    /**
+ * @example
+
+        put your example here
+
+
+ * @example
+ *
+ * ```sh
+ *
+        and the output here
+
+ * ```
+ *
+ *
+ */
+    setMessage(param1, param2, scope){
+        this.name = 'New Error';
+        this.message = `Say something about ${util.inspect(param1, {depth: 5, colors: true})} here and something about ${util.inspect(param2, {depth: 5, colors: true})}.
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/RIAEvangelist/node-error-classes/file/1941079328/issues/source?bid=3073169&fileBranchId=3104106#l29)